packages/ui/src/council/modals/RestoreVotes/RestoreVotesModal.tsx

```diff
@@ -96,7 +96,10 @@ export const RestoreVotesModal = () => {
       <ModalHeader title="Restore Votes" onClick={hideModal} />
 
       <ModalBody>
-        <p>Lorem ipsum dolor sit amet enim</p>
+        <p>
+          In case of browser storage loss or when switching devices restoring votes from a backup file may be necassary
+          for revealing. Upload your votes file created after voting here.
```

Comment:
Mostly I wanted to fix the typos on "necassary" but also I made a few changes to clarify a bit, and removed `created after voting here` because it fells to me that it implies that the file was automatically created without the user active interaction:
```suggestion
          In case of browser storage loss or when switching devices, restoring your votes from a backup file may be 
          necessary to reveal them. Upload your votes backup file here.
```

Reply:
Good point. Just had another idea to make it clear we are not uploading to a remote server. Hope that's fine.